--- a/src/models/credentials.rs
+++ b/src/models/credentials.rs
@@ -10,8 +10,6 @@
 
 use crate::models;
 use serde::{Deserialize, Serialize};
-use serde_with::serde_as;
 
-#[serde_as]
 #[derive(Clone, Default, Debug, PartialEq, Serialize, Deserialize)]
 pub struct Credentials {
     /// The URL used to access the cluster API server.
@@ -19,17 +17,14 @@ pub struct Credentials {
     pub server: Option<String>,
     /// A base64 encoding of bytes representing the certificate authority data for accessing the cluster.
     #[serde(rename = "certificate_authority_data", skip_serializing_if = "Option::is_none")]
-    #[serde_as(as = "Option<Option<_>>")]
-    pub certificate_authority_data: Option<Vec<u8>>,
+    pub certificate_authority_data: Option<String>,
     /// A base64 encoding of bytes representing the x509 client certificate data for access the cluster. This is only returned for clusters without support for token-based authentication.  Newly created Kubernetes clusters do not return credentials using certificate-based authentication. For additional information, [see here](https://docs.digitalocean.com/products/kubernetes/how-to/connect-to-cluster/#authenticate). 
     #[serde(rename = "client_certificate_data", default, skip_serializing_if = "Option::is_none")]
-    #[serde_as(as = "Option<Option<_>>")]
-    pub client_certificate_data: Option<Vec<u8>>,
+    pub client_certificate_data: Option<String>,
     /// A base64 encoding of bytes representing the x509 client key data for access the cluster. This is only returned for clusters without support for token-based authentication.  Newly created Kubernetes clusters do not return credentials using certificate-based authentication. For additional information, [see here](https://docs.digitalocean.com/products/kubernetes/how-to/connect-to-cluster/#authenticate). 
     #[serde(rename = "client_key_data", default, skip_serializing_if = "Option::is_none")]
-    #[serde_as(as = "Option<Option<_>>")]
-    pub client_key_data: Option<Vec<u8>>,
+    pub client_key_data: Option<String>,
     /// An access token used to authenticate with the cluster. This is only returned for clusters with support for token-based authentication.
     #[serde(rename = "token", skip_serializing_if = "Option::is_none")]
     pub token: Option<String>,
